src/kirigami-icons.qrc

```diff
@@ -24,5 +24,6 @@
         <file alias="icons/list-remove.svg">../3rdparty/breeze-icons/icons/actions/22/list-remove.svg</file>
         <file alias="icons/document-edit.svg">../3rdparty/breeze-icons/icons/actions/22/document-edit.svg</file>
         <file alias="icons/accept_time_event.svg">../3rdparty/breeze-icons/icons/actions/22/accept_time_event.svg</file>
+        <file alias="icons/gnumeric-formulaguru.svg">../3rdparty/breeze-icons/icons/actions/16/gnumeric-formulaguru.svg</file>
```

Comment:
i'd quite like to not be using any random picture for things... but we can swap those out later anyway

Reply:
It is not a random picture, I reviewed available images and thought that this one is more suitable for Developer mode than other, but off course we can change it later :)

Reply:
Sure, i didn't mean to say you picked an icon that was completely without reason :D What i meant is there are icons in Breeze specifically designed to suggest "development" and the like. Not sure what you're using, but just to make sure you're aware of that one, Cuttlefish is a handy little application for looking through icon sets and such, allowing you to search by name and such, for example for "develop" or "code", yielding things like this screenshot. But, at any rate, all merged and whatnot, we can work on that later, not going to be painting any bike sheds today ;)
![image](https://user-images.githubusercontent.com/5261787/53639180-6acdc100-3c20-11e9-925f-9212ff0dfb50.png)

Reply:
Yup, the `{}` is much more suitable :)